src/PDFBuilder.js

```diff
@@ -261,14 +261,11 @@ export default class PDFBuilder extends WebformBuilder {
   // 888 888     888    "Y888888 888  888  888  "Y8888        "Y8888    Y88P    "Y8888  888  888  "Y888  88888P'
   getParentContainer(component) {
     let container = [];
-    let originalComponent = null;
-    eachComponent(this.webform._form.components, (comp, path, components) => {
-      if (comp.id === component.component.id) {
-        container = components;
-        originalComponent = comp;
-        return true;
-      }
-    });
+    const originalComponent = this.webform._form.components.find(comp => comp.id === component.component.id);
```

Comment:
This will break when you have fields nested inside of panel components that have been mapped to the PDF forms.